src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = UserService.findByEmail(email);
+
+        if (user != null) {
+            return user.getPassword().equals(password);
+        }
+
         return false;
```

Comment:
let's rewrite this without using boolean literals like false
just return one logical expression